nemo/collections/nlp/metrics/classification_report.py

```diff
@@ -97,6 +97,7 @@ def get_precision_recall_f1(
         recall = torch.where(tp + fn != zeros, tp / (tp + fn) * 100, zeros)
         f1 = torch.where(precision + recall != zeros, 2 * precision * recall / (precision + recall), zeros)
 
+        print("")
```

Comment:
please don't use print(bla) - use logging instead

Reply:
It is intentionally as there was a problem in the output format like this:
Epoch 87: 100%|██████████| 184/184 [00:16<00:00, 11.05it/s, loss=0.030, v_num=0][NeMo I 2020-07-21 11:59:48 classification_report:102] label                  precision    recall       f1           support
[NeMo I 2020-07-21 11:59:48 classification_report:111] label_id: 0               93.94      93.94      93.94         33
[NeMo I 2020-07-21 11:59:48 classification_report:111] label_id: 1              100.00     100.00     100.00          9

The output log of NeMo is attached to the previous line as PT output does not print the new line.